smac/utils/io/output_writer.py

```diff
@@ -90,8 +90,11 @@ def _parse_argument(self, scenario, key: str, value):
                 except shutil.SameFileError:
                     return value  # File is already in output_dir
             elif key == 'pcs_fn' and scenario.cs is not None:
-                new_path = os.path.join(scenario.output_dir_for_this_run, 'configspace.pcs')
-                self.save_configspace(scenario.cs, new_path, 'pcs_new')
+                try:
+                    new_path = os.path.join(scenario.output_dir_for_this_run, 'configspace.pcs')
+                    self.save_configspace(scenario.cs, new_path, 'pcs_new')
+                except TypeError:
+                    traceback.print_exc()
```

Comment:
I think this output should be sent to the logger with level `ERROR`.

Reply:
that's what I tried but unfortunately, somehow at some point we try to serialize this class such that we cannot have a logger here.

Reply:
You can use this helper class to make the logger pickleable: https://github.com/automl/SMAC3/blob/development/smac/utils/logging.py